keras_rcnn/layers/object_detection/object_proposal.py

```diff
@@ -0,0 +1,59 @@
+import keras.backend
```

Comment:
But the filename `_object_proposal.py` or `object_proposal.py` makes no difference in this scenario? As long as you import the class from the `object_detection` module.